cellfinder_napari/curation.py

```diff
@@ -23,12 +31,12 @@ class CurationWidget(QWidget):
     def __init__(
         self,
         viewer: napari.viewer.Viewer,
-        cube_depth=20,
-        cube_width=50,
-        cube_height=50,
-        network_voxel_sizes=[5, 1, 1],
-        n_free_cpus=2,
-        save_empty_cubes=False,
+        cube_depth: int = 20,
+        cube_width: int = 50,
+        cube_height: int = 50,
+        network_voxel_sizes: Tuple[int, int, int] = (5, 1, 1),
+        n_free_cpus: int = 2,
+        save_empty_cubes: bool = False,
         max_ram=None,
```

Comment:
Does this have a type i.e. `Optional[int]` or something?

Reply:
I'm not sure (and it doesn't seem to be used anywhere, so I couldn't infer it 😄)

Reply:
Yeah, it's always tricky to work these things out, which is why we should really try and include typing on all new code!